standard/tier/tier_data.lua

```diff
@@ -0,0 +1,121 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Tier/Data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	tiers = {
+		{
+			order = 1,
```

Comment:
`order` isn't needed if `tiers` is an array, since it can be computed in `Module:Tier`

Reply:
tiertypes is not and i want to handle hem the same